### DISCUSSION TITLE
Type 'Time' is not assignable to type 'TimeValue | undefined' when using minValue/maxValue.

### DISCUSSION THREAD

**Discussion by pixelRena** [adobe/react-spectrum#6212] [Q&A] [answered]
Hi, 
I have the `@internationalized/date` and `react-aria-components` package installed. I'm using Typescript within my company project, but when I try to pass a `minValue={new Time(9,25)}`. I get this error from typescript, but it's exactly as shown in the documentation, so I'm not sure if I'm missing something here. I have provided the code snippet below:

```
import { TimeField, Label, DateSegment, DateInput } from 'react-aria-components'
import { Time } from '@internationalized/date'

export const Default = () => (
  <TimeField minValue={new Time(9, 25)}>
    <Label>Event time</Label>
    <DateInput>{segment => <DateSegment segment={segment} />}</DateInput>
  </TimeField>
)
```

The error: 
<img width="948" alt="image" src="https://github.com/adobe/react-spectrum/assets/98000871/393d6193-abc6-44b8-9628-adea6241365c">

**Comment by devongovett** [comment, marked as answer]
Probably you have multiple copies of `@internationalized/date` in your `node_modules` and you're trying to pass a different version than it's expecting. Try to deduplicate those.

**Reply by pixelRena** [reply]
Yep you were right! I guess somehow I may have added it again, thank you
